composer/src/main/kotlin/com/gojuno/composer/Args.kt

```diff
@@ -92,9 +94,24 @@ private class JCommanderArgs {
             names = arrayOf("--devices"),
             required = false,
             variableArity = true,
-            description = "Connected devices/emulators that will be used to run tests against. If not passed — tests will run on all connected devices/emulators. Usage example: `--devices emulator-5554 emulator-5556`."
+            description = "Connected devices/emulators that will be used to run tests against. If not passed — tests will run on all connected devices/emulators. Can't pass `--devices` with `--device-pattern`. Usage example: `--devices emulator-5554 emulator-5556`."
     )
     var devices: List<String>? = null
+
+    @Parameter(
+            names = arrayOf("--device-pattern"),
+            required = false,
+            description = "Connected devices/emulators that will be used to run tests against. If not passed — tests will run on all connected devices/emulators. Can't pass `--device-pattern` with `--devices`. Usage example: `--device-pattern \"somePatterns\"`."
```

Comment:
Let's replace "Can't pass.…" with "Specifying both `--devices` and `--device-pattern` will result in an error."